tracker-radar-tests/TR-domain-matching/domain_matching_tests.json

```diff
@@ -217,7 +217,8 @@
                 "exceptPlatforms": [
                     "web-extension",
                     "ios-browser",
-                    "web-extension-mv3"
```

Comment:
I understand that Android, as other implementations above is failing this, right? This test describes a desired behavior and at least windows is passing this one so IMO we should all align. Could you please document this like this -> https://github.com/duckduckgo/privacy-reference-tests/tree/main/fingerprinting-protections#platform-exceptions ?

Reply:
Done!